src/views/Home.vue

```diff
@@ -4,7 +4,7 @@
       <Container class="d-flex flex-items-center">
         <div class="flex-auto text-left">
           <UiButton class="pl-3 col-12 col-lg-4">
-            <Search v-model="q" placeholder="Search" />
+            <Search :value.sync="q" placeholder="Search" />
```

Comment:
Why not v-model?

Reply:
that was a fix for an error, but now it is resolved and files are updated